### ISSUE TITLE
Feedback UI Widget obscures feature to be edited

### ISSUE THREAD

**Issue by oldtopos** [mvexel/remapatron#3]
In many cases, the feature presented on the UI is obscured by the popup feedback window (yes/no, etc)  This makes matching from MR display to JOSM (or Potlatch) difficult.

**Comment by mvexel** [comment]
Agreed! I used a simple home-cooked but in no way scalable notification system initially, which I am happy to move away from.. Ideally, we should move to a more versatile and well-supported external library. I have looked at noty briefly which would fit in nicely because we're already using jQuery (for which noty is a plugin). 

Martijn van Exel

**Comment by migurski** [comment]
+1 on this.

I’d recommend ditching the popup and using space outside the map for notifications, for example a large row of large buttons across the bottom or top?

**Comment by mvexel** [comment]
Anyone here with experience implementing an existing, standard javascript notification library? Any recommendations?
